src/ripple/core/SQLInterface.md

```diff
@@ -0,0 +1,106 @@
+# SQL Interface
+
+The following changes were made to the rippled code:
```

Comment:
Please reformat to max 80 chars/line. Rational: it's easier to look at future diffs and easier to leave review comments on specific lines rather than whole paragraphs.

Reply:
Done.